source/_integrations/comfoconnect.markdown

```diff
@@ -70,15 +70,15 @@ sensor:
       - current_rmot
       - current_temperature
       - days_to_replace_filter
+      - energy_total
       - exhaust_fan_duty
       - exhaust_fan_speed
       - exhaust_humidity
       - exhaust_temperature
       - outside_humidity
       - outside_temperature
-      - power_total
       - power_usage
-      - preheater_power_total
+      - preheater_energy_total
```

Comment:
We didn't change this config item, right?

Reply:
Correct. I'll close this PR and add the note in a new pr on the current branch.